### DISCUSSION TITLE
accessing Rest Api with Keycloak JWT token

### DISCUSSION THREAD

**Discussion by Kanary159357** [Budibase/budibase#6384] [Help] [answered]
I'm trying to give rest api JWT token that has been given from keycloak auth....

But there is no way in budibase to allocate jwt in budibase.

So how can  I acheive it?

https://github.com/Budibase/budibase/discussions/4218

I've seen this issue and think this case is similar to my case but don't know how to apply it.

**Comment by melohagan** [comment, marked as answer]
Hey @Kanary159357 

I don't think it can be achieved at the moment. The discussion was more about work that needed to be done than what is currently possible.

There's an open feature request here: https://github.com/Budibase/budibase/issues/3827

**Reply by Kanary159357** [reply]
It would be just fine to get OAuth's token in budibase itself..... Thanks!

**Reply by Kanary159357** [reply]
https://github.com/Budibase/budibase/issues/4139#issuecomment-1146498982

workaround was actually here! I'll just try this
